### PR TITLE
Feature/minsu/script

### PR DESCRIPTION
**Pull request by minsulee2**
- Add ainize deploy, deposit, and request script
- Some refactoring

### CODE REVIEW

File: scripts/ainize.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  }
+
+  const job = process.argv[2];
+
+  switch (job) {
+    case 'deploy':
+      await deployService();
+      break;
+    case 'deposit':
+      // NOTE(from ainize team): deployService 에서 minCost 0으로 세팅한 경우 deposit 필요 X
```
Comment (jiyoung-an) [inline scripts/ainize.js:61]: 👍

**Review by jiyoung-an — APPROVED** [review]
LGTM, thank you for writing useful script!

File: scripts/ainize.js (hunk 1 above)
Comment (platfowner) [inline scripts/ainize.js:61]: minor) could you leave comments in English?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks for the nice job!

File: scripts/ainize.js (hunk 1 above)
Comment (minsulee2) [reply scripts/ainize.js:61]: Sure!

**Comment by minsulee2** [comment]
Thanks ! Merging now :)

**Pull request merged**